eyelid-match-ops/src/iris/conf.rs

```diff
@@ -69,49 +69,38 @@ pub type IrisCode<const STORE_ELEM_LEN: usize> = BitArray<[IrisStore; STORE_ELEM
 ///       correctly.
 pub type IrisMask<const STORE_ELEM_LEN: usize> = BitArray<[IrisStore; STORE_ELEM_LEN]>;
 
-impl IrisConf for IrisBits {
-    const COLUMN_LEN: usize = 80;
-    const COLUMNS: usize = 160;
+impl IrisConf for FullBits {
+    const COLUMNS: usize = 200;
+    const COLUMN_LEN: usize = 16 * 2 * 2;
     const ROTATION_LIMIT: usize = 15;
 }
 // TODO: work out how to automatically apply these assertions to every trait impl.
 // (Or every config type.)
 //
 // There must be enough bits to store the underlying data.
-const_assert!(IrisBits::DATA_BIT_LEN >= IrisBits::COLUMN_LEN * IrisBits::COLUMNS);
-const_assert!(IrisBits::STORE_ELEM_LEN * size_of::<IrisStore>() * 8 >= IrisBits::DATA_BIT_LEN);
+const_assert!(FullBits::DATA_BIT_LEN >= FullBits::COLUMN_LEN * FullBits::COLUMNS);
+const_assert!(FullBits::STORE_ELEM_LEN * size_of::<IrisStore>() * 8 >= FullBits::DATA_BIT_LEN);
 // Rotating more than the number of columns is redundant.
-const_assert!(IrisBits::ROTATION_COMPARISONS <= IrisBits::COLUMNS);
+const_assert!(FullBits::ROTATION_COMPARISONS <= FullBits::COLUMNS);
 // The match fraction should be between 0 and 1.
-const_assert!(IrisBits::MATCH_NUMERATOR <= IrisBits::MATCH_DENOMINATOR);
-const_assert!(IrisBits::MATCH_DENOMINATOR > 0);
+const_assert!(FullBits::MATCH_NUMERATOR <= FullBits::MATCH_DENOMINATOR);
+const_assert!(FullBits::MATCH_DENOMINATOR > 0);
 
-impl IrisConf for FullRes {
-    const COLUMN_LEN: usize = 10;
-    const COLUMNS: usize = 160;
-    const ROTATION_LIMIT: usize = IrisBits::ROTATION_LIMIT;
-}
-const_assert!(FullRes::DATA_BIT_LEN >= FullRes::COLUMN_LEN * FullRes::COLUMNS);
-const_assert!(FullRes::STORE_ELEM_LEN * size_of::<IrisStore>() * 8 >= FullRes::DATA_BIT_LEN);
-const_assert!(FullRes::ROTATION_COMPARISONS <= FullRes::COLUMNS);
-const_assert!(FullRes::MATCH_NUMERATOR <= FullRes::MATCH_DENOMINATOR);
-const_assert!(FullRes::MATCH_DENOMINATOR > 0);
-
-impl IrisConf for MiddleRes {
-    const COLUMN_LEN: usize = 5;
-    const COLUMNS: usize = 80;
-    const ROTATION_LIMIT: usize = IrisBits::ROTATION_LIMIT;
+impl IrisConf for MiddleBits {
+    const COLUMNS: usize = 100;
+    const COLUMN_LEN: usize = 8 * 2 * 2;
+    const ROTATION_LIMIT: usize = FullBits::ROTATION_LIMIT;
```

Comment:
Middle resolution iris code dimensions

Reply:
Confirmed that these are the correct dimensions for middle resolution.